forwards_paper.tex

```diff
@@ -297,17 +307,20 @@ \subsection*{Recording the pedigree in forwards time}
 the birth time, and the endpoints and parental IDs of each distinctly inherited segment,
 which are naturally stored as the \emph{nodes} and \emph{edges} of a tree sequence.
 For concreteness, here we write out in pseudocode how to run a neutral Wright--Fisher simulation
-with overlapping generations that records genealogical history in this way.
+(but with overlapping generations) that records genealogical history in this way.
 The simulation will run for $T$ generations,
 and has $N$ haploid individuals, each carrying a single chromosome of length L.
 For simplicity we sample exactly one crossover per generation.
 The probability of death per individual each generation is $\delta$.
+\jda{In the below we are using the integer vector encoding of a tree as in
+	\msprime{} should say so and reference it as the role of node ids is
+	otherwise more confusing than it need be}
```

Comment:
I don't see where the sparse tree format comes in here -- this records edges, sure, but those are described above?  What do you mean?

Reply:
I mean the fact that $P_a$ is a node id of a parent

Reply:
Right, but this is just how `edges` work?

Reply:
Er, actually - $P_j$ is the node ID of the $j$th indiv in the current generation. $a$ is the index that says "which one is my parent".